packages/lib-classifier/src/plugins/tasks/experimental/volumetric/components/VolumetricTask.js

```diff
@@ -0,0 +1,100 @@
+import asyncStates from '@zooniverse/async-states'
+import { Box, Text } from "grommet"
+import { Blank } from "grommet-icons"
+import InputStatus from "../../../components/InputStatus"
+import { Markdownz } from "@zooniverse/react-components"
+import { MobXProviderContext, observer } from "mobx-react"
+import PropTypes from "prop-types"
+import styled from "styled-components"
+import TaskInput from "../../../components/TaskInput"
+import { useContext } from 'react'
+
+// Note: ANNOTATION_COUNT will be refactored in next PR to use MobX Annotations
+const ANNOTATION_COUNT = 3
+const SVG_ARROW = "48 50, 48 15, 40 15, 50 0, 60 15, 52 15, 52 50"
+
+const StyledInstructionText = styled(Text)`
+  margin: 0;
+  padding: 0;
+  width: 100%;
+
+  > *:first-child {
+    margin-top: 0;
+  }
+`
+
+const StyledToolIcon = styled.div`
+  background-color: #2d2d2d;
+  display: flex;
+  align-items: center;
+  padding-left: 15px;
+
+  &::after {
+    content: " ";
+    margin-right: 1ch;
+    white-space: pre;
+  }
+
+  > svg {
+    height: 1.5em;
+    vertical-align: bottom;
+    width: 1.5em;
+  }
+`
+
+function VolumetricTask({ task }) {
+  const stores = useContext(MobXProviderContext)
+  const subjectReadyState = stores.classifierStore?.subjectViewer?.loadingState ?? asyncStates.loading
```

Comment:
```suggestion
function VolumetricTask({ disabled = false, task }) {
```

`disabled` is already available as a prop from Tasks
https://github.com/zooniverse/front-end-monorepo/blob/0a09c9de0186319e2823334a15eb7e0335932c90/packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/Tasks.js#L56

No need to hook into context, you should use the `disabled` prop already provided. Examples are in SurveyTask, SimpleDropdownTask, TextFromSubjectTask, SingleChoiceTask, etc.

(Sorry if this shows up as a dupe notification, I just wanted to re-comment with code changes suggested)

Reply:
Got this change incorporated as well. Thanks!